packages/web3/test/e2e/mainnet/get_past_logs.test.ts

```diff
@@ -54,11 +55,13 @@ describe(`${getSystemTestBackend()} tests - getPastLogs`, () => {
 
 	it.each(
 		toAllVariants<{
-			format: string;
+			byteFormat: string;
+			numberFormat: string;
 		}>({
-			format: Object.values(FMT_BYTES),
+			byteFormat: Object.values(FMT_BYTES),
+			numberFormat: Object.values(FMT_NUMBER),
 		}),
-	)('should getPastLogs for deployed contract', async ({ format }) => {
+	)('should getPastLogs for deployed contract', async ({ byteFormat, numberFormat }) => {
```

Comment:
is this PR related with https://app.zenhub.com/workspaces/web3js-553a980ab285bae538012a40/issues/gh/web3/web3.js/6027 ?

Reply:
as no issue is mentioned in desc

Reply:
Sorry, yes it is. It seems that the logs are being properly formatted so i just added the proper format for the tests